src/test/data/JsonModulePlannerStorageTest/invalidAndValidModuleModulePlanner.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
What does the module planner being both valid and invalid mean?

Reply:
It means the JSON file has some valid data and also some invalid data. AB's version of it was called `invalidAndValidPersonAddressBook.xml`